src/containers/ProfilePage/ToggleButton.jsx

```diff
@@ -1,23 +1,52 @@
 import React from "react";
 import { useAccordionToggle, AccordionContext } from "react-bootstrap";
+import i18n from "i18next";
 
 function getArrowClass(isCurrentElementSelected) {
-  return isCurrentElementSelected ? "arrow down" : "arrow right";
+  let arrowClass;
+
+  if (isCurrentElementSelected) {
```

Comment:
pls just make it one flat level of if-conditions, the nested if is impossible to tell whats going on

Reply:
how about this:
```
  if (isCurrentElementSelected) {
    // if Arabic, make the margin from left
    // or make it default, margin from right
    arrowClass = i18n.language === "ar"
      ? "arrow down-rtl"
      : "arrow down";
  } else if (i18n.language === "ar") {
    arrowClass = "arrow left";
  } else {
    arrowClass = "arrow right";
  }

  return arrowClass;
}
```

is using ternary operator okay? I am afraid that if would be too long and that affect the readability. @louisrli

Reply:
>  louisrli: yeah, it's fine, change it and to approve

Thank you @louisrli, I know there is no time to think ;)

Reply:
if you're going to do that it makes more sense to split it by language. naturally we think in terms fo what language the page is first and then what happens

if (language === "ar") {
 arrow = isSelected ? foo : bar
}
else {
 arrow = isSelected ?
}

Reply:
done!